addons/common/functions/fnc_debugModule.sqf

```diff
@@ -17,5 +17,5 @@
 
 params ["_entity"];
 
-GVAR(LOGDISPLAY_LEVEL) = call compile (_entity getVariable ["logDisplayLevel","4"]);
-GVAR(LOGLEVEL) = call compile (_entity getVariable ["logLevel","4"]);
+GVAR(LOGDISPLAY_LEVEL) = (parseSimpleArray format ["[%1]", _entity getVariable ["logDisplayLevel","4"]]) select 0;
+GVAR(LOGLEVEL) = (parseSimpleArray format ["[%1]", _entity getVariable ["logLevel","4"]]) select 0;
```

Comment:
PSA breaks when there are spaces.

Reply:
Yes. But there are none here. Input should be a number. Spaces in there are nonsense.